### ISSUE TITLE
pdnsutil check-zone

### ISSUE THREAD

**Issue by prowebuk** [loewexy/pdnsmanager#141]
Hi, great panel, thanks for sharing your hard work

Would be nice to add the output of "pdnsutil check-zone" to the right of the save button when viewing a domain

Many thanks

Richard

**Comment by loewexy** [comment]
In theory this would be a nice information. But I see two Problems here which prevents the implementation. First one would have to make an execute call which is hard to protect against injection attacks and opens unnecessary attack surface. The second one is that PDNS Manager must not run on the same server than Powerdns. In my own setups this is not the case. But then pdnsutil is not available to use.

**Comment by prowebuk** [comment]
Good point, off the top of my head, maybe a cron job on the master NS that monitors changes to serials in the domains table and updates a new table e.g. domains_check with the results of pdnsutil check-zone
